src/views/ReadArticle.js

```diff
@@ -252,6 +272,7 @@ export class Article extends Component {
                       className="btn delete_article"
                       onClick={() => this.handleDeleteArticle()}
```

Comment:
What do you think using references instead of a function call as we agreed in our team?